### PR TITLE
user can post comments

### PR DESCRIPTION
**Pull request by tauor**

### CODE REVIEW

**Comment by tauor** [comment]
Will remove line 35 in Comments.jsx

File: src/components/Comments/Comments.jsx [hunk 1]
```diff
@@ -21,11 +24,40 @@ const Comments = ({article_id}) => {
...
+        }
+        requests.addComment(article_id, {"username":user, "body":newComment})
+        .then((data) => {
+            setComments([data,...comments]);
+            setIsPostingComment(false);
+            setPosted(true);
+        })
+        .catch((err) => {
+            if(err.code === "ERR_NETWORK"){
+                alert('Internet connection is offline..')
```
Comment (dappernerddesigns) [inline src/components/Comments/Comments.jsx]: I'm glad you're looking at error messages here, but an alert is not the most userfriendly way of showing there is an issue. On mobile these do not display the same way as they do on desktop. Consider another way of getting this message across.

File: src/components/Comments/Comments.jsx [hunk 2]
```diff
@@ -21,11 +24,40 @@ const Comments = ({article_id}) => {
...
+        })
+    }
+
     if (loadingComments){
         return <p>Loading comments...</p>
     }else{
         return <div>
-                <ul>
+            <label htmlFor="commentInput">Comment:</label>
+            <input type="text" id="commentInput" onChange={handleChange}></input>
```
Comment (dappernerddesigns) [inline src/components/Comments/Comments.jsx]: input tags are designed for single lines of text. Our commenters may have a lot to say and want to add line breaks.

**Review by dappernerddesigns — APPROVED** [review]
I am going to approve this, I would like you to take a look at the two comments that I have left. Functionally, this is good. Take a look at some tiny amendments before you merge and move on.

**Pull request merged**